lib/sapience/extensions/grape/middleware/logging.rb

```diff
@@ -36,7 +39,7 @@ def call!(env)
               after_failure(error)
               throw(:error, error)
             else
-              @status, _, _ = *@app_response
+              @status, = *@app_response
```

Comment:
Does this actually do the same thing? I can't really understand what's happening here though. Before, it was discarding the 2nd and 3rd return values from `*@app_response`, is this syntax equivalent?